build.gradle

```
@@ -58,7 +58,7 @@ ext {
 
 ext {
     // libs
-    wordPressFluxCVersion = 'develop-fa819801570505a0e9b4f7f226bb704b88ccc1d2'
+    wordPressFluxCVersion = 'trunk-7ea7465680431fa4df5f0433d4c8b8395055b997'
```

WDYT about using the tagged version?

```suggestion
    wordPressFluxCVersion = '2.54.0'
```

---

2.54.0 is older than the trunk commit hash. I thought the process was that developers point to `trunk` and then the infra team updates it when they cut a new library release. If I'm wrong, I'd be happy to cut a new release myself.

---

You're right @0nko ! Resolving this comment.